jupyter_scheduler/tests/test_scheduler.py

```diff
@@ -39,6 +40,49 @@ def test_create_job_definition(jp_scheduler):
         assert "hello world" == definition.name
 
 
+def test_create_job_definition_with_input_folder(jp_scheduler):
+    job_definition_id = jp_scheduler.create_job_definition(
+        CreateJobDefinition(
+            input_uri="job-5/import-helloworld.ipynb",
```

Comment:
Similar to my previous comment, you can replace this with a `input_file_with_input_dir` fixture that moves `import-helloworld.ipynb` and `a/b/helloworld.txt` to `scheduler_root_dir`, and then return the tuple `('import-helloworld.ipynb', 'a/b/helloworld.txt')`.